custom_scara:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
   type: effort_controllers/JointPositionController
   joint: joint1
   pid: {p: 2.0, i: 0.09, d: 70.0}

  joint2_position_controller:
   type: effort_controllers/JointPositionController
   joint: joint2
   pid: {p: 20.0, i: 0.3, d: 110.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 4000.0, i: 0.3, d: 12.0}
